apps/twitch-bot/internal/service/service.go

```diff
@@ -108,16 +107,16 @@ func (s *services) GetGlobalBotCommand(ctx context.Context, commandName string)
 }
 
 func (s *services) GetUserBotCommand(ctx context.Context, commandName string, twitchChannelId string) (*models.BotCommand, error) {
-	commandData, err := s.DB.GetUserBotCommand(ctx, commandName, twitchChannelId)
+	commandData, err := s.DB.GetUserBotCommand(ctx, "twitch", commandName, twitchChannelId)
```

Comment:
We should also use `PlatformTwitch` or something else instead of using "twitch" as raw string throughout the code